notebooks/Imputation_best_practices/Imputation_best_practices.ipynb

```diff
@@ -0,0 +1,4557 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    m1 = sum(number_diff_1) / len(number_diff_1)
</pre><p>If we are able to compute number_diff_1 etc in vector form as in above solution, we calculating mean and variance would just be a matter of calling <code>number_diff_1.mean()</code> and <code>number_diff_1.var()</code> respectively :)   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/data-science-workflows/pull/113/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>Same as above, for loop is more convinient</p>